### PR TITLE
Discuss destruction of Pluto

### LINKED ISSUE #1: Discuss mercury
is it too close to the sun?

### PR DESCRIPTION
**Pull request by NilsPauliks**
Please accept my suggestions, I want to go home

Fixes #1

### CODE REVIEW

File: pluto.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Pluto should be destroyed
```
Comment (NilsPauliks) [inline pluto.txt:1]: Please use TNT

Comment (SowmyaMRavisandiran) [inline pluto.txt:1]: why are you so mean?

Comment (NilsPauliks) [reply pluto.txt:1]: just realistic

Comment (SowmyaMRavisandiran) [reply pluto.txt:1]: Oh well, let me merge it then

**Pull request merged**